src/terraform/terraform.ts

```diff
@@ -19,12 +20,16 @@ export class Terraform {
   public init(cwd: string, stack: TerraformStack): Deno.ChildProcess {
     const moduleFile = path.join(cwd, 'main.tf.json');
     Deno.mkdirSync(cwd, { recursive: true });
+    Deno.mkdirSync(ArcCtlConfig.getTerraformCacheDirectory(), { recursive: true });
     Deno.writeTextFileSync(moduleFile, JSON.stringify(stack.toTerraform()));
 
     return this.plugin.exec(['init', '-input=false'], {
       stdout: false,
       commandOptions: {
         cwd,
+        env: {
+          'TF_PLUGIN_CACHE_DIR': ArcCtlConfig.getTerraformCacheDirectory(),
```

Comment:
Do we need to set this, or can we inherit the value from the host machine and just document how someone can set it themselves?

Reply:
We can if it exists by using something like
```
'TF_PLUGIN_CACHE_DIR': process.env.TF_PLUGIN_CACHE_DIR || ArcCtlConfig.getTerraformCacheDirectory(),
```
Though if you want the speed increase to happen by default this has to be set here with some default value.